src/run.ts

```diff
@@ -70,6 +70,17 @@ async function run() {
         version = await getStableKubectlVersion();
     }
     let cachedPath = await downloadKubectl(version);
+
+    try {
+
+        if (!process.env['PATH'].startsWith(path.dirname(cachedPath))) {

```

Comment:
You can skip the startsWith and add it by default maybe? 

Cause the order might differ based on the order other tools are installed?

Reply:
Ok